### ISSUE TITLE
AIX - Error resolving os_patching (restrict away from AIX?)

### ISSUE THREAD

**Issue by TJM** [albatrossflavour/puppet_os_patching#93]
When testing some AIX agents (not on os_patching), we get the following error:

```
Error: Facter: error while resolving custom fact "os_patching": undefined method `+' for nil:NilClass
```

Unfortunately it doesn't give any indication of the line, but there are ways to restrict (confine) what OS's a fact tries to resolve on, right?

~tommy

**Comment by TJM** [comment]
puppet-agent-5.5.6 (PE 2018.1.4)

os_patching module version 0.6.4

Maybe we should "borrow" that template thing from voxpupuli so I remember to put all that stuff in an issue :)

**Comment by TJM** [comment]
Confirmed, the branch above (`  :ref => 'bugfix/93_confine_facter'`) is clean on AIX.

**Comment by albatrossflavour** [comment]
Thanks, I've also 'borrowed' the templates, good call.  Fixed in #98
